expenses/views/reports.py

```diff
@@ -104,8 +101,18 @@ def report_run(request, slug):
 
     if output_format == "print":
         template = "expenses/report_run_print.html"
+        content_type="text/html"
+    elif output_format == "csv":
+        template = "expenses/report_run_csv.csv"
+        content_type="text/csv"
     else:
         template = "expenses/report_run.html"
+        content_type="text/html"
+
+    report: Report = report_class(request, settings)
+    start_time = time.monotonic()
+    report_html = report.run(output_format)
```

Comment:
`report_html` is misleading; it might be better to split the run-and-render part into two functions (for CSV, without using any Django templates; and for HTML).